src/components/dashboard/SendConfirmation.web.js

```diff
@@ -0,0 +1,158 @@
+// @flow
```

Comment:
I think that we shouldn't need a platform-specific screen but just platform-specific components as `ShareButton`. This would avoid duplicating all the screen for native

Reply:
This flow is still not final, but its a good idea. keep it in mind.

Reply:
Oh... I've just uploaded the changes :(